### DISCUSSION TITLE
关于首页风格“极简”的一点建议

### DISCUSSION THREAD

**Discussion by VXleaf** [the1812/Bilibili-Evolved#1546] [功能建议 / Ideas]
### **我个人希望在“极简”首页上加一栏收藏夹内容**

- 脚本版本 ：Preview Offline v939.19
- 浏览器版本：Microsoft Edge 88.0.705.63 (官方内部版本) (64 位)

提出这个建议是因为平时上b大就是看收藏夹里的视频。。hhhh

目前“极简“首页可以显示两栏内容：“视频动态”和“热门视频”。
![image](https://user-images.githubusercontent.com/56925098/107597177-d87c2780-6c54-11eb-9b67-d5ddf512115d.png)

> 以及我很喜欢你们做的这个收藏夹下拉菜单！！如果在要在“极简”首页上添加收藏夹栏，可以模仿这个样式做吗qwq
> ![image](https://user-images.githubusercontent.com/56925098/107597313-3872ce00-6c55-11eb-86d9-3fa13fbc8ec6.png)

如果这部分工作实现起来很麻烦的话。。那大大就当我没说吧hhhhh现在这个样子已经很满意了。

**祝大大新年快乐**

**Comment by the1812** [comment]
新年快乐, 如果每次都看收藏夹的内容的话, 直接进空间里看就行了: `https://space.bilibili.com/你的UID/favlist`
